desktop/settings/linux-settings.md

```diff
@@ -0,0 +1,151 @@
+---
+description: Docker Desktop settings
+keywords: Settings, Docker Desktop, Linux
+title: Change Docker Dektop settings on Linux
+---
+
+This page provides information on how to configure and manage your Docker Desktop settings.
+
+To navigate to **Settings** either:
+
+- Select the Docker menu ![whale menu](../images/whale-x.png){: .inline} and then **Settings** 
+- Select the **Settings** icon from the Docker Dashboard. 
+
+## General
+
+On the **General** tab, you can configure when to start Docker and specify other settings:
+
+- **Start Docker Desktop when you log in**. This automatically starts Docker Desktop when you open your session.
+
+- **Send usage statistics**. Docker Desktop sends diagnostics, crash reports, and usage data. This information helps Docker improve and troubleshoot the application. Clear the check box to opt out.
+
+- **Show weekly tips**. Select to display useful advice and suggestions about using Docker.
+
+- **Open Docker Desktop dashboard at startup**. Select to automatically opens the dashboard when starting Docker Desktop.
+
+- **Use Docker Compose V2**. Select to enable the `docker-compose` command to use Docker Compose V2. For more information, see [Docker Compose V2](../../compose/index.md#compose-v2-and-the-new-docker-compose-command)
+
+## Resources
+
+The **Resources** tab allows you to configure CPU, memory, disk, proxies, network, and other resources.
+
+#### Advanced
+
+On the **Advanced** tab, you can limit resources available to Docker.
+
+Advanced settings are:
+
+- **CPUs**. By default, Docker Desktop is set to use half the number of processors
+available on the host machine. To increase processing power, set this to a
+higher number; to decrease, lower the number.
+
+- **Memory**. By default, Docker Desktop is set to use 25% of your host's memory. To increase the RAM, set this to a higher number. To decrease it, lower the number.
+
+- **Swap**. Configure swap file size as needed. The default is 1 GB.
+
+- **Disk image size**. Specify the size of the disk image.
+
+- **Disk image location**. Specify the location of the Linux volume where containers and images are stored.
+
+You can also move the disk image to a different location. If you attempt to move a disk image to a location that already has one, you are asked if you want to use the existing image or replace it.
+
+### File sharing
+
+Use File sharing to allow local directories on the Linux host to be shared with Linux containers.
```

Comment:
(re: common content); if we rephrase this to (e.g.) "on your machine" (instead of "on the Linux host"), the content is the same between all platforms.